src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.ts

```diff
@@ -470,14 +471,42 @@ export class ExerciseAssessmentDashboardComponent implements OnInit, AfterViewIn
      * Calculates the status of a submission by inspecting the result
      * @param submission Submission which to check
      */
-    calculateStatus(submission: Submission) {
+    calculateSubmissionStatus(submission: Submission) {
         const tmpResult = submission.latestResult;
         if (tmpResult && tmpResult!.completionDate && Result.isManualResult(tmpResult!)) {
             return 'DONE';
         }
         return 'DRAFT';
     }
 
+    calculateComplaintStatus(complaint: Complaint) {
+        // a complaint is handled if it is either accepted or denied and a complaint response exists
+        const handled = complaint.accepted !== undefined && complaint.complaintResponse !== undefined;
```

Comment:
Should the check be `!complaint.accepted && complaint.complaintResponse !== undefined`? Because if the complaint isn't accepted and the complaint response exists, then it will be counted as handled